app/tests/Unit/Components/SessionDisplayTest.php

```diff
@@ -42,6 +42,16 @@
     expect($view)->toSeeText('Parliament is meeting in Strasbourg this week.');
 });
 
+it('displays link to the agenda of the session', function () {
+    $view = $this->blade('<x-session-display :currentSession="$currentSession" :lastSession="$lastSession" :nextSession="$nextSession" />', [
+        'currentSession' => $this->currentSession,
+        'lastSession' => $this->lastSession,
+        'nextSession' => $this->nextSession,
+    ]);
+
+    expect($view)->toSee('https://www.europarl.europa.eu/doceo/document/OJ-9-2021-08-01-SYN_EN.html');
```

Comment:
Maybe we can be a little more specific?

```suggestion
    expect($view)->toHaveSelector('a[href="https://www.europarl.europa.eu/doceo/document/OJ-9-2021-08-01-SYN_EN.html"]', text: 'agenda');
```